### PR TITLE
Query returns null for the image field of the likedBy object of the post. 

### PR DESCRIPTION
**Pull request by prayanshchh**
**What kind of change does this PR introduce?**
bugfix

**Issue Number:**
#2262
Fixes #

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
image

**If relevant, did you update the documentation?**
No

**Summary**
This solves the issue of null image of users that had liked posts.

**Does this PR introduce a breaking change?**
No, it's a bug fix

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**
yes

## Summary by CodeRabbit

- **New Features**
	- Enhanced post retrieval by including additional user data related to the `likedBy` field, such as user images, first names, and last names, improving the richness of information available to users.
- **Bug Fixes**
	- Improved formatting and clarity in installation and documentation files for better user experience.
- **Tests**
	- Introduced new unit tests for post-related resolvers to ensure correct functionality of the like feature and data retrieval.
- **Configuration Changes**
	- Adjusted configuration settings for improved test execution performance and consistency in formatting.

### CODE REVIEW

**Comment by prayanshchh** [comment]
@palisadoes ??

**Comment by prayanshchh** [comment]
@pranshugupta54 can you please review it sir?

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please add a test to ensure that this error doesn't repeat itself.

**Comment by prayanshchh** [comment]
@palisadoes I have added a test, this is the first time I have written a test so if you have any practices that I should follow so that i can update this and keep that in mind for my next PR's, thank you sir.

File: .coderabbit.yaml
Comment (palisadoes) [inline .coderabbit.yaml:18]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

File: INSTALLATION.md
Comment (palisadoes) [inline INSTALLATION.md:327]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

File: README.md
Comment (palisadoes) [inline README.md:49]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

File: scripts/cloud-api-demo/README.md
Comment (palisadoes) [inline scripts/cloud-api-demo/README.md:239]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

File: vite.config.mts
Comment (palisadoes) [inline vite.config.mts:55]: Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please make the suggested changes

**Comment by prayanshchh** [comment]
making a new pr

**Comment by prayanshchh** [comment]
yeah i am on it

**Comment by prayanshchh** [comment]
am really sorry for this, I will remember this next time, won't happen again
again my apologies

**Comment by prayanshchh** [comment]
the reviews suggested by coderabbit.ai are outdated as the files have been unstaged

**Comment by palisadoes** [comment]
@coderabbit.ai

**Comment by palisadoes** [comment]
@coderabbitai

**Comment by prayanshchh** [comment]
sir don't merge this PR, when I started the admin I saw this error
image

after seeing this I also included _id in select of populate but it still showed the same error, but the api shows no error like this

**Comment by prayanshchh** [comment]
Hi @palisadoes I have fixed all errors related to the PR, I just need some help to fix this linting error, as I have a failing test, I tried checked with prettier but it showed everything was good, I even ran `npx prettier --write ` but all the files were unchanged

can you please tell me what am I missing here

**Comment by palisadoes** [comment]
Please ask the #talawa-api slack channel for assistance

**Comment by prayanshchh** [comment]
I actually did ask this on slack, but nobody replied, that's why I tagged you, I will ask this again on slack.

**Comment by palisadoes** [comment]
@pranshugupta54 @varshith257 Do you have any suggestions?

**Review by palisadoes — APPROVED** [review]

**Pull request merged**
